store/storetest/team_store.go

```diff
@@ -115,7 +115,10 @@ func testTeamStoreUpdateDisplayName(t *testing.T, ss store.Store) {
 		t.Fatal(err)
 	}
 
-	ro1 := (<-ss.Team().Get(o1.Id)).Data.(*model.Team)
+	ro1, err := ss.Team().Get(o1.Id)
+	if err != nil {
```

Comment:
nit: We can replace this patter for `require.Nil(t, err)` which is more concise and expressive.